### PR TITLE
Make offline players fetch their UUID from Mojang if the server is in online mode

### PR DESCRIPTION
**Pull request by gdude2002**
Related to #227 and perhaps #365

### CODE REVIEW

**Comment by gdude2002** [comment]
Builds: http://bamboo.gserv.me/browse/GLOW-SRV-PR409-79 (obviously)

File: src/main/java/net/glowstone/io/nbt/NbtPlayerDataService.java [hunk 1]
```diff
@@ -86,6 +88,17 @@ public UUID lookupUUID(String name) {
                 return player.getUniqueId();
             }
         }
+
+        // If we're in online mode, let's find the real damn UUID already
+        if (Bukkit.getServer().getOnlineMode()) {
```
Comment (Johni0702) [inline src/main/java/net/glowstone/io/nbt/NbtPlayerDataService.java]: This should probably also check for active proxy support and fetch the UUID if it's enabled using `GlowServer.getProxySupport()`.

**Comment by turt2live** [comment]
PR testing completed:

Assuming the player has never joined before...
- GOOD JOB Opping players on an offline-mode server ops the player's offline UUID without warning.
- GOOD JOB Opping players on an online-mode server ops the player's real UUID without warning (for valid players).
- GOOD JOB Opping players on an online-mode server who do not exist anywhere ops the player's offline UUID and issues a warning.

Assuming the player has played before...
- GOOD JOB Opping players whoa re currently online on the server are opped correctly (by offline or online UUID depending on server state)

Notes:
- To ensure that #365 is correctly resolved, when this is pulled the ticket author will have to verify functionality. 
- Other commands were tested such as whitelist and ban to ensure operation.

**Comment by gdude2002** [comment]
Checkstyles is being dumb about constants, can we fix this?

**Comment by turt2live** [comment]
@gdude2002 Your PR does not compile! Here's some information about the compile testing:
- Glowkit PR found: `false`
- Glowkit PR parsed: `_NONE_`
- Glowkit compiled: `false`

If your PR requires Glowkit, please ensure that an accurate link to the Glowkit PR appears in the description of your PR. If the information above is incorrect, please contact me on IRC (`turt2live` in `#glowstone` on esper.net) for more information.

If your PR does not require Glowkit, please fix your PR.

Thank you!

`PR_BOT_CODE:3` (you can ignore this)

**Comment by gdude2002** [comment]
Are we about ready to call your script broken, @turt2live?

**Comment by turt2live** [comment]
Yes.

**Comment by turt2live** [comment]
Pulled via https://github.com/GlowstoneMC/Glowstone/commit/88e3c28945f6cb58b8877441d51e10050fc9ab4b

**Pull request closed without merging**